core/commands/files.go

```diff
@@ -1002,6 +1002,7 @@ Remove files or directories.
 	},
 	Options: []cmdkit.Option{
 		cmdkit.BoolOption("recursive", "r", "Recursively remove directories."),
+		cmdkit.BoolOption("force", "Allow to remove anything else"),
```

Comment:
What do you think of `forcibly remove target at path; implies -r for directories` as a description?